meetings/2025-05-20.md

```diff
@@ -0,0 +1,43 @@
+# W3C Docs CG call, 2025-05-20
+
+Present:
+
+Chair: Florian
+
+Note taker:
+
+Joining info:
+
+To join the video meeting, click this link: https://meet.google.com/jdp-xvka-hyk
+Otherwise, to join by phone, dial +49 40 8081617306 and enter this PIN: 544 514 889#
+To view more phone numbers, click this link: https://tel.meet/jdp-xvka-hyk?hs=5
+
+## Agenda
+
+1. Welcoming more new folks
+2. Web Component docs follow-up
+    - Issue: https://github.com/w3c-cg/webdocs/issues/1
+    - Meeting notes https://github.com/w3c-cg/webdocs/blob/main/meetings/2025-05-06.md
+    - Any more thoughts since we met two weeks ago?
+3. Web Security Docs
+    - Will to present current status and outline for future work
+5. Documentation feedback sources
+    - What are sources of feedback for web platform docs?
+        - Doc bugs filed
+        - MDN satisfaction data
+        - State of XYZ surveys (this was very useful for the Web Components docs discussion!)
+    - User interviews
+        - Would attendees of this CG be willing to be interviewed?
+6. Preparing topics for the next few Docs CG calls
+    - (Florian) I'm interested in meeting up with Immersive Web CG to discuss [WebXR docs](https://github.com/openwebdocs/project/issues/158)
+    - (Florian) Also interested in meeting up with Performance WG to discuss [Performance API docs](https://github.com/openwebdocs/project/issues/157)
+    - What else?
+7. _please add your items_
```

Comment:
```suggestion
7. (Christine): Suggest how WC3 can possibly fill the void the [Season of Docs program has left for individuals who are struggling to get paid tech documentation experience](https://github.com/w3c-cg/webdocs/issues/5)
```

Reply:
Check out my idea @Elchi! :)

Reply:
Please see my answer in your issue: https://github.com/w3c-cg/webdocs/issues/5#issuecomment-2889953893